lib/CMakeLists.txt

```diff
@@ -100,7 +113,7 @@ if(WIN32)
     set_target_properties(${LIB_NAME} PROPERTIES IMPORT_SUFFIX "_imp.lib")
 
     set_target_properties (${LIB_NAME} PROPERTIES
-       DEBUG_POSTFIX "-d"
+       DEBUG_POSTFIX "d"
```

Comment:
i realize having just built using cmake that DEBUG_POSTFIX adds -d to every pe it builds so for example there's lib123-d.exe etc. I think those should be lib123.exe and the only thing that needs the d suffix is curld and libcurld, that way they are like the projects filenames.

/cc @webmaster128

Reply:
I personally see no value in this suffixes as we use separate build folders for debug builds (which is cmake default I guess). So I have no opinion on this one as long as I can disable all suffixes.

Reply:
@jay I think the postfix for exe is set somewhere else. I'll check it out.

@webmaster128 the postfixes have value when you install the project. You can install both flavours into one directory what simplifies path handling for projects that use curl.

Reply:
@webmaster128 CMake separate build dirs are required on Linux, because of CMake and ninja/make limitations.
But typically you build both release and debug in the same build tree on windows, because you can switch between the build types within MSVC.
You do get separate bin subdirs within the build dir, but typically they all get installed to the same dir.

I personally prefer postfix for .exes, because you only use the debug exes if you are debugging something.  Else you use the release version, so its good to know you are stepping away from the norm.

Reply:
and -1 for the change from -d to d, see #2121 for my 2 cents